### PR TITLE
Camera: Reset using screenspace

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

**Comment by mwestphal** [comment]
The only change that looks strange is : testing/baselines/TestInteractionFocalPointPickingPoints.png , but I checked it and it is fine, focal point picking happens on another point.

File: library/src/camera_impl.cxx [hunk 1]
```diff
@@ -202,7 +209,23 @@ camera& camera_impl::resetToDefault()
...
 {
+
+#if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 0, 20210331)
   this->Internals->VTKRenderer->ResetCamera();
+#else
+#if F3D_MODULE_EXTERNAL_RENDERING
+  if (vtkExternalOpenGLRenderWindow::SafeDownCast(
+        this->Internals->VTKRenderer->GetRenderWindow()) != nullptr)
+  {
+    // External render window does not support ResetCameraScreenSpace correctly
```
Comment (Meakk) [inline library/src/camera_impl.cxx:220]: why?

File: library/src/camera_impl.cxx [hunk 2]
```diff
@@ -202,7 +209,23 @@ camera& camera_impl::resetToDefault()
 //----------------------------------------------------------------------------
 camera& camera_impl::resetToBounds()
 {
+
+#if VTK_VERSION_NUMBER < VTK_VERSION_CHECK(9, 0, 20210331)
   this->Internals->VTKRenderer->ResetCamera();
+#else
+#if F3D_MODULE_EXTERNAL_RENDERING
+  if (vtkExternalOpenGLRenderWindow::SafeDownCast(
```
Comment (Meakk) [inline library/src/camera_impl.cxx:217]: Calling `IsA` might be better, what do you think?

Comment (mwestphal) [reply library/src/camera_impl.cxx:217]: definitely

File: library/src/camera_impl.cxx (hunk 1 above)
Comment (mwestphal) [reply library/src/camera_impl.cxx:220]: no idea, I tried tracking it down but no luck and did not want to spend too much time on it. Using ResetCameraScreenSpace results on an empty image even though the computed bounds looks fine.

**Review by Meakk — APPROVED** [review]

**Pull request merged**